### PR TITLE
Remove clone_uri

### PR DESCRIPTION
**Pull request by mkumatag**
This is not required anymore because this repo is public now and does not need ssh way of closing it.

/assign @yussufsh @Rajalakshmi-Girish

### CODE REVIEW

**Comment by ltccci** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/ppc64le-cloud/test-infra/pull/151#" title="Author self-approved">mkumatag</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=ppc64le-cloud%2Ftest-infra).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[config/jobs/ocp-automation/OWNERS](https://github.com/ppc64le-cloud/test-infra/blob/master/config/jobs/ocp-automation/OWNERS)~~ [mkumatag]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by ltccci** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/ppc64le-cloud/test-infra/pull/151#" title="Author self-approved">mkumatag</a>*, *<a href="https://github.com/ppc64le-cloud/test-infra/pull/151#pullrequestreview-608767404" title="LGTM">sudeeshjohn</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=ppc64le-cloud%2Ftest-infra).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[config/jobs/ocp-automation/OWNERS](https://github.com/ppc64le-cloud/test-infra/blob/master/config/jobs/ocp-automation/OWNERS)~~ [mkumatag]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Review by bpradipt — APPROVED** [review]
/lgtm

**Comment by ltccci** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/ppc64le-cloud/test-infra/pull/151#pullrequestreview-608769319" title="LGTM">bpradipt</a>*, *<a href="https://github.com/ppc64le-cloud/test-infra/pull/151#" title="Author self-approved">mkumatag</a>*, *<a href="https://github.com/ppc64le-cloud/test-infra/pull/151#pullrequestreview-608767404" title="LGTM">sudeeshjohn</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=ppc64le-cloud%2Ftest-infra).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[config/jobs/ocp-automation/OWNERS](https://github.com/ppc64le-cloud/test-infra/blob/master/config/jobs/ocp-automation/OWNERS)~~ [mkumatag]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
